FinSearchUnified/Resources/services/services.xml

```diff
@@ -6,16 +6,31 @@
 
     <services>
         <service
-            id="fin_search_unified.color_picker_facet_result"
-            class="FinSearchUnified\Bundles\FacetResult\ColorPickerFacetResult"
+                id="fin_search_unified.color_picker_facet_result"
```

Comment:
The indentations seem a little bit off. Make sure that the attributes are on the same level as `argument` or `tag`.